.circleci/config.yml

```diff
@@ -0,0 +1,43 @@
+version: 2
+jobs:
+  checkout_cuberite:
+    working_directory: ~/cuberite
+    docker:
+    - image: peterbell10/cuberite-circleci:0.0.2
```

Comment:
Can we auto-deploy this image to the docker hub? Or at least to a place, where multiple users could update the image?
I'm mainly thinking about ways to update the image via pull requests...

Reply:
Seems like [it's possible](https://docs.docker.com/docker-hub/builds/). In fact [cuberite/docker-ci](https://github.com/cuberite/docker-ci) might even be doing something similar already.